lib/web_ui/test/path_test.dart

```diff
@@ -536,5 +536,19 @@ void testMain() {
       end = iter.skipToNextContour();
       expect(start, end);
     });
+
+    /// Regression test for https://github.com/flutter/flutter/issues/68702.
+    test('Path should return correct bounds after transform', () {
+      final Path path1 = Path()
+        ..moveTo(100, 100)
+        ..lineTo(200, 100)
+        ..lineTo(150, 200)
+        ..close();
+      final SurfacePath path2 = Path.from(path1) as SurfacePath;
+      Rect bounds = path2.pathRef.getBounds();
+      SurfacePath transformedPath = path2.transform(
+          Matrix4.identity().scaled(0.5, 0.5).toFloat64());
+      expect(bounds != transformedPath.pathRef.getBounds(), isTrue);
```

Comment:
```suggestion
      expect(transformedPath.pathRef.getBounds(), isNot(bounds));
```

Instead of a "expected true but was false" test failure, you can get a slightly more explicit error message if you use isNot(equals()) or similar, at least you get the toString of the objects that were being compared :P

Reply:
Done. Thank you for suggestion!